### PR TITLE
[#51384] fixed storage admin pages

### PR DESCRIPTION
**Pull request by Kharonus**
[OP#51384](https://community.openproject.org/wp/51384)

### Wat?

- synch display texts
- updated static links
- hide "incomplete" status labels, if storage is pristine (initial state on new storage)
- remove default values "consumers" and "My storage"

### CODE REVIEW

File: modules/storages/app/controllers/storages/admin/storages_controller.rb [hunk 1]
```diff
@@ -85,7 +85,7 @@ def select_provider
     service_result = ::Storages::Storages::SetAttributesService
                  .new(user: current_user,
                       model: @object,
-                      contract_class: Storages::Storages::BaseContract,
+                      contract_class: EmptyContract,
                       contract_options: { skip_provider_type_strategy: true })
```
Comment (akabiru) [inline modules/storages/app/controllers/storages/admin/storages_controller.rb]: _nit:_ We  no longer need to [skip_provider_type_strategy](https://github.com/opf/openproject/blob/dev/modules/storages/app/contracts/storages/storages/base_contract.rb#L51) if we've switched from `Storages::Storages::BaseContract`
```suggestion
                      contract_class: EmptyContract)
```

File: modules/storages/app/components/storages/admin/storage_view_information.rb [hunk 2]
```diff
@@ -23,6 +23,9 @@ def configuration_check_label
     end
 
     def configuration_check_label_for(*configs)
+      # do not show the status label, if storage is completely empty (initial state)
+      return nil if storage.configuration_checks.values.none?
```
Comment (akabiru) [inline modules/storages/app/components/storages/admin/storage_view_information.rb]:
```suggestion
      return if storage.configuration_checks.values.none?
```

File: modules/storages/app/components/storages/admin/storage_view_information.rb [hunk 3]
```diff
@@ -35,6 +38,9 @@ def status_label(label, scheme:, test_selector:)
     end
 
     def automatically_managed_project_folders_status_label
+      # do not show the status label, if storage is completely empty (initial state)
+      return nil if storage.configuration_checks.values.none?
```
Comment (akabiru) [inline modules/storages/app/components/storages/admin/storage_view_information.rb]:
```suggestion
      return if storage.configuration_checks.values.none?
```

File: modules/storages/spec/features/admin_storages_spec.rb [hunk 4]
```diff
@@ -291,6 +293,9 @@
         aggregate_failures 'General information' do
           within_test_selector('storage-general-info-form') do
             fill_in 'storages_one_drive_storage_name', with: 'My OneDrive', fill_options: { clear: :backspace }
+            fill_in 'storages_one_drive_storage_tenant_id',
+                    with: '029d4741-a4be-44c6-a8e4-e4eff7b19f65',
+                    fill_options: { clear: :backspace }
```
Comment (akabiru) [inline modules/storages/spec/features/admin_storages_spec.rb]: _nit_  we shouldn't need to clear the input if we no longer have a default storage name

```suggestion
                    with: '029d4741-a4be-44c6-a8e4-e4eff7b19f65'
```

**Review by akabiru — APPROVED** [review]
Nice one @Kharonus 🥇 LGTM a few non-blocking recommendations

----

⚠️ Do you want to base against `release/13.1`?

File: modules/storages/app/controllers/storages/admin/storages_controller.rb (hunk 1 above)
Comment (Kharonus) [reply modules/storages/app/controllers/storages/admin/storages_controller.rb]: removed

File: modules/storages/app/components/storages/admin/storage_view_information.rb (hunk 2 above)
Comment (Kharonus) [reply modules/storages/app/components/storages/admin/storage_view_information.rb]: removed

File: modules/storages/app/components/storages/admin/storage_view_information.rb (hunk 3 above)
Comment (Kharonus) [reply modules/storages/app/components/storages/admin/storage_view_information.rb]: removed

File: modules/storages/spec/features/admin_storages_spec.rb (hunk 4 above)
Comment (Kharonus) [reply modules/storages/spec/features/admin_storages_spec.rb]: removed

**Comment by Kharonus** [comment]
Thanks @akabiru ,

rebased to `release/13.1` (good find) and fixed the issues you pointed out.

**Review by akabiru — COMMENTED** [review]
:shipit:

**Pull request merged**